src/processor.ts

```diff
@@ -87,18 +92,64 @@ export class ProcReference implements ProcHints, InputReference {
     return this.formatContributors(this.author);
   }
 
-  // write a method to render to string from the AST
+  // does this belong here, or on Processor?
+  getTemplate(key: string): InlineTemplate {
```

Comment:
Aside from answering the question of whether to move it, also need "cannot find template" error handling.